fe/fe-core/src/main/java/com/starrocks/catalog/FunctionSet.java

```diff
@@ -542,11 +542,11 @@ private void initAggregateBuiltins() {
             addBuiltin(AggregateFunction.createBuiltin(name,
                     Lists.newArrayList(Type.INT), Type.BIGINT, Type.BIGINT, false, true, false));
             addBuiltin(AggregateFunction.createBuiltin(name,
-                    Lists.newArrayList(Type.DECIMAL32), Type.DECIMAL64, Type.DECIMAL64, false, true, false));
+                    Lists.newArrayList(Type.DECIMAL32), Type.DECIMAL128, Type.DECIMAL128, false, true, false));
```

Comment:
decimal 64 couldn’t cover the decimal 32 sum result?

Reply:
max precision of decimal32 is 9, max precision of decimal64 is 18, so result of sum on billions of decimal32  can exceeds decimal64.